apps/graphql/src/apps/booking/types/outputs/BookingInterface.js

```diff
@@ -66,6 +69,38 @@ export const commonFields = {
     type: GraphQLBookingType,
     description: 'OneWay, Multicity or Return',
   },
+  segmentLocations: {
+    type: GraphQLList(GraphQLCoordinate),
+    description:
+      'This will contain the gps-coordinates for all stops on the booking',
```

Comment:
In the same way we have `deprecated` flags, would it be possible to have some kind of `future feature` flag? In my opinion, a description should use present tense, but I understand how it is necessary here if there is no such `feature` flag.

Reply:
Yeah, you are right, I will change it to `This contains ...`

Reply:
🚀